### PR TITLE
[15.0][IMP] website_form_require_legal: Set link to terms and conditions

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
Improve the template by allowing to set the URL when rendering the qweb template by setting a value for the termsURL variable by context.

cc @Tecnativa TT40305

@pedrobaeza @chienandalu please review

### CODE REVIEW

File: website_form_require_legal/static/src/js/options.js [hunk 1]
```diff
@@ -35,7 +35,6 @@ odoo.define("website_form_require_legal.form_editor", function (require) {
                 $(template).html(
                     qweb.render("website_form_require_legal.s_website_form_legal", {
                         labelWidth: labelWidth,
-                        termsURL: "terms",
```
Comment (pedrobaeza) [inline website_form_require_legal/static/src/js/options.js:38]: Why removing this default value?

Comment (pedrobaeza) [reply website_form_require_legal/static/src/js/options.js:38]: I confirm that this is working with a sane default `/terms`, so please restore it with that value.

Comment (pilarvargas-tecnativa) [reply website_form_require_legal/static/src/js/options.js:38]: done!

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**